_posts/2021/2021-09-16-5-most-valuable-things-i-learned-in-my-10-plus-year-career.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: "5 most valuable things I learned in my 10+ year career"
+date: 2021-09-16
+header_image: public/5-things-learning.jpg
+header_overlay: true
+header_position: center
+category: tech-stories
+tags: ["coding tips", "learning", "development", "soft skills"]
+authors: ["Waldemar"]
+about_authors: ["wschneider"]
+---
+
+Back in late 2008, I started to work as a Junior Java Developer at a middle-sized company in the finance sector.
+Today at ePages, but overall I've been working as a developer (mostly with JVM-based languages) for almost 13 years.
+Along the journey, I came across many different types of people, organizations, cultures, and obviously software.
+With this post, I want to present the 5 (for me) most valuable things I learned in my 10+ year career.
+Let's break it down.
+
+
+### 1. Emotional intelligence
+
+Many young developers want to become better at coding.
+But looking back, coding was the simpler task compared to dealing with conflicts, people, or problems you need to solve.
+At this point, emotional intelligence is very useful.
+You have probably heard this term before.
+But what does it mean, you may ask?
+
+Basically, it consists of 4 parts:
+1. Perceiving emotions: The ability to realize the emotions of people.
+This includes facial expressions, gestures, posture and voice.
+2. Using emotions: The ability or knowledge to use your and other people's emotions to ease cognitive activities, e.g. problem or conflict solving.
+3. Understanding: The ability to analyze emotions and how these can change over time.
+4. Managing: The ability to influence the emotions of yourself or other people.
+
+
+
+### 2. Think about 'The Big Picture'
+
+The big picture can have a lot of meanings, depending on the person you ask.
+While working on projects or products, usually you become the expert of your team's domain.
+Many developers tend to overestimate the importance of their own domain in the system.
+
+Let's assume there are 3 views on that:
+- Business: How to be more successful as a company, and how can this be achieved?
+- User: How can we provide value to the end-user or customer with the code you write?
```

Comment:
"How can we... with the code we write" or "How can you... with the code you write", but the subject of the sentence should stay the same.